alacritty_terminal/src/input.rs

```diff
@@ -414,9 +414,17 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
             prev_line != self.ctx.mouse().line || prev_col != self.ctx.mouse().column;
         let mouse_moved = cell_changed || prev_side != cell_side;
 
+        if !mouse_moved {
```

Comment:
I think we can just inline the `mouse_moved` code here.

Reply:
Sounds good!